### PR TITLE
Add AliMeeting multi-condition training recipe

### PR DESCRIPTION
**Pull request by desh2608**
This is similar to the AMI ASR recipe, i.e., we train a single model by combining training data from different mic settings: IHM, IHM + reverb, SDM, and GSS-enhanced audio, and use this model to evaluate different conditions.

This recipe also includes text normalization similar to the M2MeT challenge baseline. Here are the results using `--epoch 15 --avg 8` and modified beam search:

| Evaluation set           | eval CER    | test CER |
|--------------------------|------------|---------|
| IHM                      |  9.58  | 11.53 |
| SDM                      |  23.37  | 25.85 |
| MDM (GSS-enhanced)       |  11.82  | 14.22 |

As a comparison, the eval set results for single-speaker ASR from the [baseline paper](https://arxiv.org/pdf/2110.07393.pdf) are:

image

In the above table, the `Eval-Ali-near` is equivalent to IHM, and `Eval-Ali-far` is equivalent to SDM.

### CODE REVIEW

File: egs/alimeeting/ASR_v2/RESULTS.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        --avg 8 \
+        --exp-dir ./pruned_transducer_stateless5/exp \
+        --max-duration 500 \
+        --decoding-method fast_beam_search \
+        --beam 4 \
+        --max-contexts 4 \
+        --max-states 8
+```
+
+Pretrained model is available at <https://huggingface.co/desh2608/icefall-asr-alimeeting-pruned-transducer-stateless7>
```
Comment (csukuangfj) [inline egs/alimeeting/ASR_v2/RESULTS.md:87]: Could you upload some test_waves to this huggingface repo?
It makes testing the model easier in sherpa.

Please have a look at
https://huggingface.co/csukuangfj/icefall-asr-librispeech-pruned-transducer-stateless8-2022-11-14/tree/main/test_wavs

Comment (desh2608) [reply egs/alimeeting/ASR_v2/RESULTS.md:87]: Done.

Comment (desh2608) [reply egs/alimeeting/ASR_v2/RESULTS.md:87]: If there are no further changes needed, perhaps this can be merged?

**Review by csukuangfj — APPROVED** [review]

File: egs/alimeeting/ASR_v2/RESULTS.md (hunk 1 above)
Comment (csukuangfj) [reply egs/alimeeting/ASR_v2/RESULTS.md:87]: Thanks!

**Comment by csukuangfj** [comment]
By the way, I have added the model to
https://huggingface.co/spaces/k2-fsa/automatic-speech-recognition

Screen Shot 2022-12-10 at 6 25 25 PM

File: egs/alimeeting/ASR_v2/RESULTS.md (hunk 1 above)
Comment (csukuangfj) [reply egs/alimeeting/ASR_v2/RESULTS.md:87]: @desh2608 You can try your pre-trained model at https://huggingface.co/spaces/k2-fsa/automatic-speech-recognition

Thanks again for the contribution.

**Comment by desh2608** [comment]
> By the way, I have added the model to https://huggingface.co/spaces/k2-fsa/automatic-speech-recognition
> 
> Screen Shot 2022-12-10 at 6 25 25 PM

Thanks! I'll check it out.

**Comment by csukuangfj** [comment]
@desh2608 
Could you please merge
https://huggingface.co/desh2608/icefall-asr-alimeeting-pruned-transducer-stateless7/discussions/1

(Your test waves are 32-bit encoded using floats. I converted it to 16-bit encoded using int16 in the above PR).

**Comment by desh2608** [comment]
> @desh2608 Could you please merge https://huggingface.co/desh2608/icefall-asr-alimeeting-pruned-transducer-stateless7/discussions/1
> 
> (Your test waves are 32-bit encoded using floats. I converted it to 16-bit encoded using int16 in the above PR).

Thanks. Done.
